apps/fares/lib/proposed_locations.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Fares.ProposedLocations do
+  @moduledoc """
+    Gets information from the existing ArcGIS data related to proposed sales locations, parses the json and returns it.
+  """
+
+  alias Fares.ProposedLocations.Location
+  require Logger
+
+  @base_url "https://services1.arcgis.com/ceiitspzDAHrdGO1/ArcGIS/rest/services/ProposedSalesNetworkSpringOutreach/FeatureServer/0/query?f=json&outFields=*&inSR=4326&outSR=4326&returnGeometry=true"
+
+  @distance_in_miles 2
```

Comment:
This might be too small a limit for the more suburban areas.... would it make sense to do a very large limit and limit the results shown to the (nine? I don't know) closest ones?

Reply:
Totally -- I'll change that.